### DISCUSSION TITLE
Welcome to ThinkPad T460s macOS OpenCore Discussions!

### DISCUSSION THREAD

**Discussion by simprecicchiani** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mhl221135** [comment]
I install ubuntu on dedicated hdd
And its recomend to update bios (can i update? Mac os with your efi would be working?)
And also its recoment to update uefi device firmware (what is this ?)
![MVIMG_20210118_214944.jpg](https://user-images.githubusercontent.com/55281358/105098656-13a29500-5ab3-11eb-853c-344c22bc3b3c.jpg)

**Comment by mhl221135** [comment]
Is there a chance to make "hiberinatemode 25" work with your EFI?I try via pmset but led on poweron button still blinking like on sta
maybe i need to change some in OC folder?
I changed hiberinatemode to AUTO in config

**Comment by 559911** [comment]
Thank you for your work！
I’m using dual system on my t460s with one ssd，MacOS works great but under the windows10 my computer was recognized as ‘MacBook pro’ and the software Lenovo vantage can’t update my driver(because of the wrong model type and serial number ) is there any way to fix it?
I find that if I boot from the boot menu(press f12 and choose my ssd)then win10 has a right model .

**Reply by simprecicchiani** [reply]
To prevent SMBIOS infos to get passed to Windows set:
- `Kernel -> Quirks -> CustomSMBIOSGuid` : `True`
- `PlatformInfo -> UpdateSMBIOSMode` : `Custom`
Unfortunately, this doesn't make Lenovo Vantage to work.

Out of curiosity, how did you installed Windows and managed to have both EFI partition showing up on BIOS Boot Menu?

**Comment by Kagayaki23** [comment]
Hi 
I want to know, what is the BIOS mod for? Does it remove the Lenovo logo when you first boot up? And what is the impact, especially for windows?

Thank you

**Reply by simprecicchiani** [reply]
The BIOS mod unlock the advanced tab from which you can tweak power and performance settings. Definitely not a big deal.

**Reply by Kagayaki23** [reply]
is it like overclocked?

**Reply by simprecicchiani** [reply]
No, Intel U-series are not overclockable.

**Reply by Kagayaki23** [reply]
![IMG_20210222_223630](https://user-images.githubusercontent.com/61573021/108794199-ea948900-75b7-11eb-8697-6453dfa8e4c7.jpg)
can i modify the BIOS in windows? If so, which first step should I take?

**Reply by simprecicchiani** [reply]
https://notthebe.ee/Removing-the-Wi-Fi-Whiteslit-on-Haswell-Thinkpads-T440p-W540-T540-etc.html
